backend/vcdat/FlaskTests.py

```diff
@@ -1,10 +1,11 @@
-from app import app
-import unittest
-import os
+if True:  # pragma: no cover
+    from app import app
+    from Files import isValidFile
+    import unittest
+    import os
 
 
-class FlaskTests(unittest.TestCase):
-
+class FlaskTests(unittest.TestCase):  # pragma: no cover
```

Comment:
You shouldn't need to cover the tests in `pragma: no cover`; you should write a bash script that passes appropriate filters to `coverage.py` to prevent it from measuring any files that have "test" in the name. Also, technically, all of our tests should be in a subpackage "test", which would prevent any of this from being a problem. Or, we can have a tests/ directory at the root of the package that would hold all of the backend and frontend tests. Any of those are better options than the way this is set up right now, which is super hacky and nonstandard.